tests/bindings.lisp

```diff
@@ -101,7 +101,8 @@
   (let ((*foreign-library-directories* (list (load-directory))))
     (load-foreign-library 'libtest)
     (load-foreign-library 'libtest2)
-    #+fsbv (load-foreign-library 'libfsbv)
+    (when (cffi-feature-p :struct-by-value)
+      (load-foreign-library 'libfsbv))
```

Comment:
I don't understand what that `#+fsbv` was doing there since `cffi-tests` depends on `cffi-libffi` unconditionally. I think we can just load libfsbv unconditionally.